src/hooks/useCioAutocomplete.ts

```diff
@@ -37,7 +38,7 @@ const useCioAutocomplete = (options: UseCioAutocompleteOptions) => {
 
   const [query, setQuery] = useState('');
   const previousQuery = usePrevious(query);
-  const cioClient = useCioClient({ apiKey, cioJsClient } as CioClientConfig);
+  const cioClient = useCioClient({ apiKey, cioJsClient, cioJsClientOptions } as CioClientConfig);
```

Comment:
Since API key is a cioJsClientOption. What do you think about deprecating `apiKey` as a standalone parameter and make it passed as part of `cioJsClientOptions`? Or is it better to just leave it separate?

Reply:
I think it's better to leave it separate. I feel like most of the customers won't use `cioJsClientOptions` unless they are making a hybrid integration. I think `apiKey` makes it way easier to understand and get up an running with the library

Reply:
Yeah probably it makes more sense to library consumers that don't necessarily need to know that this is a config for the client